buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -0,0 +1,488 @@
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "a9db38f9-9fdc-478c-b0f9-464221e58316"
+      },
+      "inputs": {
+        "workItemType": "234347",
+        "assignToRequestor": "true",
+        "additionalFields": "{}"
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "57578776-4c22-4526-aeb0-86b6da17ee9c"
+      },
+      "inputs": {
+        "additionalFields": "{}"
+      }
+    }
+  ],
+  "variables": {
+    "system.debug": {
+      "value": "false",
+      "allowOverride": true
+    },
+    "PB_ConfigurationGroup": {
+      "value": "Release"
+    },
+    "PB_RepoUrl": {
+      "value": "http://github.com/dotnet/core-setup.git"
+    },
+    "PB_GitDirectory": {
+      "value": "/root/core-setup"
+    },
+    "PB_DockerContainerName": {
+      "value": "$(Build.BuildId)"
+    },
+    "PB_DockerImageName": {
+      "value": "$(PB_DockerRepository):$(PB_DockerTag)"
+    },
+    "PB_DockerRepository": {
+      "value": "chcosta/dotnetcore"
+    },
+    "PB_DockerTag": {
+      "value": "ubuntu1404_cross_prereqs_v1",
+      "allowOverride": true
+    },
+    "PB_CloudDropAccountName": {
+      "value": "dotnetbuildoutput"
+    },
+    "CloudDropAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "OfficialBuildId": {
+      "value": "$(Build.BuildNumber)"
+    },
+    "PB_Label": {
+      "value": "$(Build.BuildNumber)"
+    },
+    "GitHubBranch": {
+      "value": "sni_plus_latestbuildtools"
+    },
+    "PB_Architecture": {
+      "value": "arm"
+    },
+    "SourceVersion": {
+      "value": "HEAD",
+      "allowOverride": true
+    },
+    "PB_DockerCopyDest": {
+      "value": "$(Build.BinariesDirectory)/docker_repo"
+    },
+    "portableLinux": {
+      "value": ""
+    },
+    "CrossToolsetVersion": {
+      "value": "",
+      "allowOverride": true
+    },
+    "SkipUnmount": {
+      "value": "--SkipUnmount"
+    },
+    "TargetRid": {
+      "value": "ubuntu.14.04-arm",
+      "allowOverride": true
+    }
+  },
+  "demands": [
+    "Agent.OS -equals linux"
+  ],
+  "retentionRules": [
+    {
+      "branches": [
+        "+refs/heads/*"
+      ],
+      "artifacts": [],
+      "artifactTypesToDelete": [
+        "FilePath",
+        "SymbolStore"
+      ],
+      "daysToKeep": 10,
+      "minimumToKeep": 1,
+      "deleteBuildRecord": true,
+      "deleteTestResults": true
+    }
+  ],
+  "buildNumberFormat": "$(date:yyyyMMdd)$(rev:-rr)",
+  "jobAuthorizationScope": "projectCollection",
+  "jobTimeoutInMinutes": 90,
+  "repository": {
+    "properties": {
+      "labelSources": "0",
+      "reportBuildStatus": "true",
+      "fetchDepth": "0",
+      "gitLfsSupport": "false",
+      "skipSyncSource": "false",
+      "cleanOptions": "0"
+    },
+    "id": "58fa2458-e392-4373-ba2b-dd3ef0c2d7ce",
+    "type": "TfsGit",
+    "name": "DotNet-CoreFX-Trusted",
+    "url": "https://devdiv.visualstudio.com/DevDiv/_git/DotNet-CoreFX-Trusted",
```

Comment:
Why does this build definition use CoreFX as its repository instead of Core-Setup like the other build definitions?  I was expecting the whole `repository` node to be identical to the other build definitions.  I think doing so would also eliminate the need for the "git clone" and "git checkout" steps and the `PB_RepoUrl` variable.